addons/mail/controllers/thread.py

```diff
@@ -125,24 +125,20 @@ def read_subscription_data(self, follower_id):
         record.check_access("read")
         # find current model subtypes, add them to a dictionary
         subtypes = record._mail_get_message_subtypes()
-        followed_subtypes_ids = set(follower.subtype_ids.ids)
-        subtypes_list = [
-            {
-                "name": subtype.name,
-                "res_model": subtype.res_model,
-                "sequence": subtype.sequence,
-                "default": subtype.default,
-                "internal": subtype.internal,
-                "followed": subtype.id in followed_subtypes_ids,
-                "parent_model": subtype.parent_id.res_model,
-                "id": subtype.id,
-            }
-            for subtype in subtypes
-        ]
-        return sorted(
-            subtypes_list,
-            key=lambda it: (it["parent_model"] or "", it["res_model"] or "", it["internal"], it["sequence"]),
-        )
+        return {
+            "store_data": Store(
+                subtypes,
+                ["default", "internal", "name", "parent_id", "res_model", "sequence"],
```

Comment:
`parent_id` should return its `res_model` field possibly?
How did we use `parent_model` previously?

Is `parent_id` even used at all?